brion/csvConfig.cpp

```diff
@@ -29,36 +30,14 @@
 
 namespace
 {
-// Taken from:
-// https://stackoverflow.com/questions/890164/how-can-i-split-a-string-by-a-delimiter-into-an-array
-std::vector<std::string> explode(const std::string& str, const char& ch)
+std::vector<std::string> explode_whitespace(const std::string& str)
 {
-    std::string next;
-    std::vector<std::string> result;
-
-    // For each character in the string
-    for (auto it = str.begin(); it != str.end(); it++)
-    {
-        // If we've hit the terminal character
-        if (*it == ch)
-        {
-            // If we have some characters accumulated
-            if (!next.empty())
-            {
-                // Add them to the result vector
-                result.push_back(next);
-                next.clear();
-            }
-        }
-        else
-        {
-            // Accumulate the next character into the sequence
-            next += *it;
-        }
-    }
-    if (!next.empty())
-        result.push_back(next);
-    return result;
+    const boost::char_separator<char> sep{" "};
```

Comment:
Could you add additional whitespace in the test file to test that the tokenizer drops empty tokens between whitespaces.